docs/data/external-data/ga-srtm/_data.yaml

```diff
@@ -0,0 +1,61 @@
+# See the Product metadata fields documentation:
+# https://docs.dev.dea.ga.gov.au/public_services/dea_knowledge_hub/product_metadata_fields.html
+
+# Overview
+
+title: SRTM 1 second DEM
+long_title: SRTM 1 second DEM version 1.0
+header_image: /_files/external-data/ga_srtm_thumbnail.jpg
+product_type: null
+spatial_data_type: Raster
+time_span:
+  start: 2014
+  end: 2014
+update_frequency: null
+next_update: null
+product_ids:
+  - ga_srtm_dem1sv1_0
+
+doi: null
+ecat: "72759"
+nci: null
+licence:
+  name: Creative Commons Attribution 4.0 International Licence
+  link: https://creativecommons.org/licenses/by/4.0/
+
+citations:
+  data_citation: "Gallant, J., Wilson, N., Dowling, T., Read, A., Inskeep, C. 2011. SRTM-derived 1 Second Digital Elevation Models Version 1.0. Record 1. Geoscience Australia, Canberra."
+  paper_citation: null
+custom_citations: null
+
+tags:
+  - digital elevation model
+  - srtm
+
+# Access
+
+external_data_page:
+  link: https://pid.geoscience.gov.au/dataset/ga/72759
+  custom_label: "See an example of how to use this data in the DEA Notebook: `Extracting contour lines </notebooks/How_to_guides/Contour_extraction/>`_"
```

Comment:
@robbibt  Ive added a link to the relevant DEA Notebook https://knowledge.dea.ga.gov.au/notebooks/How_to_guides/Contour_extraction/
to this product: https://pr-238-preview.khpreview.dea.ga.gov.au/data/external-data/ga-srtm/?tab=access

Reply:
Awesome idea - I'd forgotten we used it in that notebook! Just checking though - did you mean to update the "External data page" link? I really liked the original tile, and I'm not sure we should overload it with too much extra functionality - I reckon this would be better done simply as a new "Code example" tile (matching how we do it on our main DEA product pages).

Reply:
Totally agree. Done